ims-apple/IMS Apple/Views/AddingProduct/AddingProductView.swift

```diff
@@ -0,0 +1,172 @@
+//
+//  AddingProductView.swift
+//  IMS Apple
+//
+//  Created by Diana Zeledon on 13/8/24.
```

Comment:
NIT: Update the file name to `Brandon Santeliz` pls.

```suggestion
//  Created by Brandon Santeliz on 13/8/24.
```

Reply:
No needed